ebms-send-in/src/main/kotlin/no/nav/emottak/ebms/App.kt

```diff
@@ -85,30 +92,44 @@ fun Application.ebmsSendInModule() {
                                             request.addressing.replyTo(request.addressing.service, it.msgInfo.type.v),
-                                            utbetalingXmlMarshaller.marshalToByteArray(it)
+                                            UtbetalingXmlMarshaller.marshalToByteArray(it)
                                         )
                                     }
                                 }
-                            else ->
-                                timed(appMicrometerRegistry, "frikort-sporing") {
-                                    frikortsporring(wrapMessageInEIFellesFormat(request)).let {
-                                        SendInResponse(
-                                            request.messageId,
-                                            request.conversationId,
-                                            request.addressing.replyTo(
-                                                it.eiFellesformat.mottakenhetBlokk.ebService,
-                                                it.eiFellesformat.mottakenhetBlokk.ebAction
-                                            ),
-                                            frikortXmlMarshaller.marshalToByteArray(it.eiFellesformat.msgHead)
-                                        )
-                                    }
+
+                            "HarBorgerEgenandelFritak", "HarBorgerFrikort" -> timed(
+                                appMicrometerRegistry,
+                                "frikort-sporing"
+                            ) {
+                                frikortsporring(wrapMessageInEIFellesFormat(request)).let {
+                                    SendInResponse(
+                                        request.messageId,
+                                        request.conversationId,
+                                        request.addressing.replyTo(
+                                            it.eiFellesformat.mottakenhetBlokk.ebService,
+                                            it.eiFellesformat.mottakenhetBlokk.ebAction
+                                        ),
+                                        FellesFormatXmlMarshaller.marshalToByteArray(it.eiFellesformat.msgHead)
+                                    )
                                 }
+                            }
+
+                            "PasientlisteForesporsel" -> timed(appMicrometerRegistry, "PasientlisteForesporsel") {
+                                if (isProdEnv()) {
+                                    throw NotImplementedError("PasientlisteForesporsel is used in prod. Feature is not ready. Aborting.")
+                                }
+                                PasientlisteService.pasientlisteForesporsel(request)
+                            }
+
+                            else -> {
+                                throw NotImplementedError("Service: ${request.addressing.service} is not implemented")
```

Comment:
Hva blir konsekvens ved å kaste exception, i stedet for å bare logge direkte og gjøre ingenting?